### DISCUSSION TITLE
Add Chromium Embedded Framework

### DISCUSSION THREAD

**Discussion by PARTOVIY** [SmartlyDressedGames/Unturned-3.x-Community#4088] [Ideas]
Interface developers for Unturned are severely limited by the inability to add Unity Script to their interfaces.
I think this problem can be solved by adding CEF support.

https://cefsharp.github.io/

**Comment by GazziFX** [comment]
Look at #1950

**Comment by GazziFX** [comment]
https://ultralig.ht/
An alternative embedding framework aimed at game engines.

P.S. it doesn't have official Unity integration.
UI ToolKit and client-side uScript should fit better
